### PR TITLE
Turn off EAP capability for RTC workflow

### PR DESCRIPTION
**Pull request by seongsujeong**
This is to turn off the EAP capability on `s1-reader` to avoid AUX_CAL loading.
Just a line of change, but we need `s1_orbit.get_orbit_file_from_list()`, in order to run this without error. The most recent version of `s1-reader` is missing this function. I've suggested to revive the functionality in the PR [here](https://github.com/opera-adt/s1-reader/pull/79)

### CODE REVIEW

**Review by gshiroma — COMMENTED** [review]
LGTM. Before approving and merging it to the main repo, do we need any other changes to be able to use the latest version of the S-1 reader?

**Comment by seongsujeong** [comment]
> LGTM. Before approving and merging it to the main repo, do we need any other changes to be able to use the latest version of the S-1 reader?

No change would be necessary on RTC to use `s1-reader` as long as my PR on the reader [here]( https://github.com/opera-adt/s1-reader/pull/79) is merged. The current version of `s1-reader` does not have `get_orbit_file_from_list()`, which RTC is using in `runconfig.py`. The PR will revive the function and we can run `rtc-s1.py` as it is.

**Comment by gshiroma** [comment]
> > LGTM. Before approving and merging it to the main repo, do we need any other changes to be able to use the latest version of the S-1 reader?
> 
> No change would be necessary on RTC to use `s1-reader` as long as my PR on the reader [here](https://github.com/opera-adt/s1-reader/pull/79) is merged. The current version of `s1-reader` does not have `get_orbit_file_from_list()`, which RTC is using in `runconfig.py`. The PR will revive the function and we can run `rtc-s1.py` as it is.

That's awesome! Let's wait for that PR to get merged then.

**Review by gshiroma — APPROVED** [review]

**Pull request merged**
